src/components/Registration/index.jsx

```diff
@@ -0,0 +1,97 @@
+import React, {useState, useEffect} from 'react';
+import './style.css';
+import userIcon from './img/user-solid.svg'
+
+const Registation = () => {
+
+    const [user, setUser] = useState({
+        email: "",
+        username: "",
+        password: "",
+        passwordConfirm: "",
+      });
+
+    const handleSubmit= (e) => {
+        e.preventDefault();
+        if (user.password === user.passwordConfirm && user.username !== '' && user.email !== '' && user.password !== '' && user.passwordConfirm !== '') {
```

Comment:
[ArcanisCz](https://github.com/ArcanisCz) [7 days ago](https://github.com/ArcanisCz/registrace-zadani/pull/1#discussion_r977441792)

kdybys to chtěla zkrátit, dá se udělat i něco jako:
if([user.username, user.email].every((value) => value !== "")
;)
reálně bys na takovéhle věci použila pak nějakou validační knihovnu protože dělat to ručně je vopruz https://www.cssscript.com/best-javascript-form-validator/ (a nebo jen html 5 validace https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation)